net-p2p/qbittorrent/qbittorrent-9999.ebuild

```diff
@@ -56,28 +56,22 @@ BDEPEND+="
 
 DOCS=( AUTHORS Changelog {CONTRIBUTING,README}.md )
 
-src_prepare() {
-	MULTIBUILD_VARIANTS=()
-	use gui && MULTIBUILD_VARIANTS+=( gui )
-	use webui && MULTIBUILD_VARIANTS+=( nogui )
-
-	cmake_src_prepare
-}
-
 src_configure() {
+	MULTIBUILD_VARIANTS+=( $(usev gui) $(usev webui nogui) )
+
 	my_src_configure() {
 		local mycmakeargs=(
-			# musl lacks execinfo.h
-			-DSTACKTRACE=$(usex !elibc_musl)
-			# More verbose build logs are preferable for bug reports
-			-DVERBOSE_CONFIGURE=ON
-			-DWEBUI=$(usex webui)
+			-DVERBOSE_CONFIGURE=ON # for bug reports
+			-DSTACKTRACE=$(usex !elibc_musl) # musl lacks execinfo.h
 			-DTESTING=$(usex test)
+			-DWEBUI=$(usex webui)
```

Comment:
- I'm not really sure putting the comments at the end instead of above the relevant line helps here.
- if you're moving the order around, I'd expect that to be e.g. for sorting alphabetically, but it's... not?